sessions/backend-create/demo-end/pages/api/jokes/[id].js

```diff
@@ -0,0 +1,17 @@
+import dbConnect from "../../../db/connect";
+import Joke from "../../../db/models/Joke";
+
+export default async function handler(request, response) {
+  await dbConnect();
+  const { id } = request.query;
+
+  if (request.method === "GET") {
+    const joke = await Joke.findById(id);
+
+    if (!joke) {
+      return response.status(404).json({ status: "Not Found" });
+    }
+
+    response.status(200).json(joke);
+  }
```

Comment:
else case is missing and should respond with a 501 `response.status(501).json({error: "not implemented"})` or `response.status(501).end()`

Reply:
I'd like to implement this in every other instance as well, hence why I'd like to open an issue for this as well. :)